README.md

```diff
@@ -12,7 +12,7 @@ A [Terraform](https://www.terraform.io) module to create and manage a Google
 Kubernetes Engine (GKE) cluster with autopilot enabled.
 
 **_This module supports Terraform version 1
-and is compatible with the Terraform Google Provider version 4._**
+and is compatible with the Terraform Google Provider Beta version ~> 4.34**
```

Comment:
we should think about how to communicate the stuff we discussed yesterday about beta and non beta setups...(just a note)